syncmaster/backend/api/v1/transfers.py

```diff
@@ -45,6 +45,7 @@ async def read_transfers(
     group_id: int,
     page: int = Query(gt=0, default=1),
     page_size: int = Query(gt=0, le=200, default=20),
+    search_query: str | None = Query(None, title="Search Query", description="full-text search for transfers"),
```

Comment:
```suggestion
    search_query: str | None = Query(None, title="Search Query", description="full-text search for transfers", deferred=True),
```
Нет смысла читать этот вектор из БД, достаточно использовать его только в WHERE